webroot/results/includes/_database.php

```diff
@@ -166,7 +166,7 @@ function getAllUsedCountriesCompetitions () {
 #----------------------------------------------------------------------
 function getAllUsedContinents () {
 #----------------------------------------------------------------------
-  return dbQuery("SELECT * FROM Continents");
+  return dbQuery("SELECT * FROM Continents WHERE recordName <> ''");
```

Comment:
So, this got a little complicated. We don't want to show "Multiple Continents" on the [records page](https://staging.worldcubeassociation.org/results/events.php), but we *do* want to show it on the [competitions search](https://staging.worldcubeassociation.org/competitions/). I'm sure we're going to forget about this in the future, so I'm very open to removing the Multiple Continents thing now, or later when it proves to be a PITA.